src/bika/lims/browser/analysisrequest/add2.py

```diff
@@ -1806,6 +1806,12 @@ def ajax_submit(self):
                     msg = _("Contact does not belong to the selected client")
                     fielderrors["Contact"] = msg
 
+            # Auto-add CCContact when hidden on Sample Add form
+            if not record['CCContact'] and contact_obj:
```

Comment:
Note that the record always contains a key for the `CCContact` field, even when the field is hidden. As a result, if the field is visible in the Add Sample form and the user manually clears its value, the condition will still evaluate to `True`. In that case, the system will automatically set the value for `CCContact`, even though the user explicitly removed them during entry.

To avoid this, you should check whether the field is hidden rather than whether it has a value. Hint: you can retrieve the field with `self.get_field("CCContact")` and determine hidden fields with `self.get_fields_with_visibility("hidden", mode="add")`.